File: mario.c
#include <stdio.h>
#include <cs50.h>


int main(void)
{
    int h, w, k, l, m, i;
    
     i = get_int("Height: ");

    
    for (h = i*2;h > i;h--)
    {
        for (w = h; w > i+1;w--)
        {
            printf(" ");
        }
        for (k= h-i;k < i+1;k++)
        {
            printf("#");
        }
        for (l = i-1;l < i;l++)
        {
            printf("  ");
        }
        for (m = h-i;m < i+1;m++)
        {
            printf("#");
        }
        
        
        
        printf("\n");
    }

}

File: python-version/mario.py


from cs50 import get_int


while True:
    n = get_int("Positive number: ")
    if n > 0:
        break

for i in range(n):

    for j in range(n-i):
        print(" ", end="")
    for k in range(i+1):
        print("#", end="")
    print()
